extensions/sql-database-projects/src/controllers/mainController.ts

```diff
@@ -37,7 +37,7 @@ export default class MainController implements vscode.Disposable {
 	public deactivate(): void {
 	}
 
-	public async activate(): Promise<IProjectProvider> {
+	public async activate(): Promise<SqlDatabaseProjectProvider> {
```

Comment:
Was this an oversight before that the return type was IProjectProvider?

Reply:
it was fine to return IProjectProvider before since SqlDatabaseProjectProvider only had the methods in the IProjectProvider interface. Now that SqlDatabaseProjectProvider also implements IExtension, we have to return SqlDatabaseProjectProvider here so that the methods from both interfaces can be used.